app/assets/javascripts/account/reports.js

```diff
@@ -0,0 +1,83 @@
+document.addEventListener('turbolinks:load', () => {
+  var graphs = document.getElementById('graphs');
+
+  if(graphs) {
+    var ids = graphs.querySelectorAll('*[id]');
```

Comment:
this is not ok 
you checking ALL elements with "id"
what if I've something else with id and it's not a chart
use classes and explicitly  specify elements which you want to use for charts